src/plugins/matrixops/eye_operation.cpp

```diff
@@ -44,7 +44,10 @@ namespace phylanx { namespace execution_tree { namespace primitives {
             "Returns:\n"
             "\n"
             "Return an N x M matrix with ones on the k-th diagonal and zeros\n"
-            "elsewhere")};
+            "elsewhere)",
+            true
```

Comment:
If you use a raw string (`R"(…)"`) you don't have to use quotes and '\n' inside the actual string.